src/components-examples/material/tabs/tab-group-harness/tab-group-harness-example.html

```diff
@@ -0,0 +1,12 @@
+<mat-tab-group>
+  <mat-tab label="First" aria-label="First tab">
```

Comment:
Rather than using names like "First", "Second", "Content 1", etc., could you use some contrived content? IMO, using meaningful examples makes them easier to internalize.